woocommerce/woocommerce-android
# TPP: Hide illustration on the reader type selection dialog in landscape

Closes: #8208


### Description

The PR hide illustration in the landscape mode on the reader-type selection dialog. We do the same for all TPP/IPP related dialog

### Testing instructions

* Run TPP flow
* Notice that in the landscape mode the layout of the readers type selection dialog looks good now

### Images/gif



https://user-images.githubusercontent.com/4923871/213658370-86423b01-73e1-4b81-a743-aa6b91328469.mp4
